packages/backend/src/open_payments/quote/service.ts

```diff
@@ -239,7 +240,7 @@ export async function startQuote(
       quoteOptions.amountToSend = options.sendAmount.value
     } else {
       quoteOptions.amountToDeliver =
-        options.receiveAmount?.value || options.receiver.incomingAmount.value
+        options.receiveAmount?.value || options.receiver.incomingAmount?.value
```

Comment:
https://github.com/interledgerjs/interledgerjs/tree/master/packages/pay#quoteoptions
>Either amountToSend, amountToDeliver, or destinationPayment must be provided

Are we ok with `Pay` throwing instead of us checking that here?

Reply:
Well, it is already in a try/catch environment and Pay checks anyway. Do you think we need another check?

Reply:
probably ok to just have Pay check